workshop/posting-error-example.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Posting an error to Slack
+---
+<p><img style = "padding: 0 15px; float: left;" src = "screenshots/slack-cancer-data-science-logo.png" width = "75"></p>
+<p style="margin-top: 20px;"> </p>
+<p>We use the <b>Cancer Data Science Slack</b> for communication.
+If you haven't yet joined Cancer Data Science Slack, you will need to follow the [set up procedures described here](../virtual-setup/slack-procedures.md).
+
+During and after the workshop, we encourage you to post your coding questions to your Slack training channel.
+But, asking questions about code in such a way that others can readily help you can be tricky and is a skill itself!
+
+Here we've laid out guidelines for posting your question so that your peers and the CCDL team will readily be able to help you find a solution.
+Following these guidelines will take more time but it will increase your chances of getting speedier responses that more adequately help you resolve your error!
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [What information should a question about code include?](#what-information-should-a-question-about-code-include)
+- [A troubleshooting example](#a-troubleshooting-example)
+  - [Step 1) Do what you can to try to solve the error yourself (but don't tire yourself out!)](#step-1-do-what-you-can-to-try-to-solve-the-error-yourself-but-dont-tire-yourself-out)
+  - [Step 2) Draft out your question you'll post.](#step-2-draft-out-your-question-youll-post)
+    - [Where is this error occurring; what notebook/chunk?](#where-is-this-error-occurring-what-notebookchunk)
+    - [What is the goal of this code?](#what-is-the-goal-of-this-code)
+    - [What is the code that is producing this error?](#what-is-the-code-that-is-producing-this-error)
+    - [What is the error or problematic outcome?](#what-is-the-error-or-problematic-outcome)
+    - [(Optional) What have you tried thus far?](#optional-what-have-you-tried-thus-far)
```

Comment:
```suggestion
      - [Where is this error occurring; what notebook/chunk?](#where-is-this-error-occurring-what-notebookchunk)
      - [What is the goal of this code?](#what-is-the-goal-of-this-code)
      - [What is the code that is producing this error?](#what-is-the-code-that-is-producing-this-error)
      - [What is the error or problematic outcome?](#what-is-the-error-or-problematic-outcome)
      - [(Optional) What have you tried thus far?](#optional-what-have-you-tried-thus-far)
```
I think that's required for this to render with the indentation properly (at least in the markdown)

Reply:
I just trusted doctoc (perhaps foolishly).